Frontend/src/pages/register/components/RegistrationEditPanel.jsx

```diff
@@ -13,18 +13,16 @@ export default function RegistrationEditPanel({ registration }) {
   const [comment, setComment] = useState(registration.comment)
   const { competition_id } = useParams()
   const { isLoading, heldEvents } = useHeldEvents(competition_id)
-
-  const handleEventSelection = (selectedEvents) => {
-    setSelectedEvents(selectedEvents)
-  }
   return isLoading ? (
     <div className={styles.panel}>
       <LoadingMessage />
     </div>
   ) : (
     <div className={styles.panel}>
       <EventSelector
-        handleEventSelection={handleEventSelection}
+        handleEventSelection={(selectedEvents) =>
```

Comment:
Sorry if I was unclear, I meant `handleEventSelection={setSelectedEvents}` - no need to create an anonymous function at all.

Reply:
ah of course, changed.